azureLogIngestion/index.ts

```diff
@@ -6,11 +6,10 @@ const apiKey = process.env["NEW_RELIC_INSERT_KEY"]
 
 const adapter = new Adapter(apiKey)
 
-const eventHubTrigger: AzureFunction = async function (context: Context, eventHubMessages: any[]): Promise<void> {
+const eventHubTrigger: AzureFunction = async function (context: Context, eventHubMessages: any): Promise<void> {
```

Comment:
Oddly, the event hub trigger template that `func` creates expects `eventHubMessages` to be an array. Should figur eout why you're not seeing an array.

Reply:
Yeah we do get an array but its `eventhubmessage.requests` as far as I could find. But if this turns out to not be the case down the road we can change it.